src/Event/ReadModel/JSONLD/EventLDProjector.php

```diff
@@ -544,8 +544,13 @@ protected function applyEventCopied(
      */
     protected function applyEventDeleted(EventDeleted $eventDeleted)
     {
-        $this->repository->remove($eventDeleted->getItemId());
-        return null;
+        $document = $this->loadDocumentFromRepository($eventDeleted);
```

Comment:
@LucWollants Om dubbele code te vermijden tussen `EventLDProjector` en `PlaceLDProjector` zou ik `AbstractOfferDeleted` toepassen in de `OfferLDProjector`.

Reply:
Zeker een terechte opmerking en ik had het eerst ook generisch geïmplementeerd. Daarna heb ik toch 3 aparte implementaties genomen omdat het dan makkelijker wordt om afwijkend gedrag te implementeren.